android/src/main/java/com/swmansion/rnscreens/gamma/tabs/TabsHostViewManager.kt

```diff
@@ -179,6 +179,13 @@ class TabsHostViewManager :
         view?.tabBarItemTitleFontSizeActive = value
     }
 
+    override fun setTabBarItemRippleColor(
```

Comment:
Please add @ReactProp annotation here with `customType = "Color"`, otherwise it won't work with legacy architecture.